src/Apps/Sell/Components/SubmissionHeader.tsx

```diff
@@ -14,6 +14,7 @@ import createLogger from "Utils/logger"
 import { useFormikContext } from "formik"
 import { useState } from "react"
 
+const HEADER_HEIGHT = 40
```

Comment:
I'm setting the height, because now we use a mix of buttons and links which have different heights.